### ISSUE TITLE
Size limits

### ISSUE THREAD

**Issue by psergiu** [dosdude1/2.5-inch-ide-ssd#1]
Any way to limit the SSD size to a lower than actual value (137GB, 8GB, 2GB, 528MB) for compatibility with older systems ?
See: https://tldp.org/HOWTO/Large-Disk-HOWTO-4.html
Thanks !

**Comment by dosdude1** [comment]
Yes, when programming with the MPTool, there is an option you can set to specify the usable size (to as low as 256MB). Though of course you'd want to use relatively small NANDs as to not waste larger ones (you can have 1, 2 or 4 NANDs populated). You can also specifically set the cylinder, head, and sector count if needed.
